src/test/resources/pages/instructorFeedbackResultsSortRecipientGiverQuestion.html

```diff
@@ -3550,7 +3550,7 @@
           <tbody>
             <tr>
               <td>
-                <i>Instructors</i>
+                &lt;i&gt;Instructors&lt;/i&gt;
```

Comment:
it don't think it is trivial to determine if a html tag found in the text is from the user or our own inline styling tags.